### PR TITLE
Updating workflows/microbiome/gene-based-pathogen-identification from 0.1 to 0.2

### PR DESCRIPTION
**Pull request by gxydevbot**
Hello! This is an automated update of the following workflow: **workflows/microbiome/gene-based-pathogen-identification**. I created this PR because I think one or more of the component tools are out of date, i.e. there is a newer version available on the ToolShed.

By comparing with the latest versions available on the ToolShed, it seems the following tools are outdated:
* `toolshed.g2.bx.psu.edu/repos/bgruening/split_file_to_collection/split_file_to_collection/0.5.0` should be updated to `toolshed.g2.bx.psu.edu/repos/bgruening/split_file_to_collection/split_file_to_collection/0.5.2`
* `toolshed.g2.bx.psu.edu/repos/bgruening/flye/flye/2.9.1+galaxy0` should be updated to `toolshed.g2.bx.psu.edu/repos/bgruening/flye/flye/2.9.3+galaxy0`
* `toolshed.g2.bx.psu.edu/repos/iuc/medaka_consensus_pipeline/medaka_consensus_pipeline/1.7.2+galaxy0` should be updated to `toolshed.g2.bx.psu.edu/repos/iuc/medaka_consensus_pipeline/medaka_consensus_pipeline/1.7.2+galaxy1`
* `toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_find_and_replace/1.1.4` should be updated to `toolshed.g2.bx.psu.edu/repos/bgruening/text_processing/tp_find_and_replace/9.3+galaxy1`

The workflow release number has been updated from 0.1 to 0.2.

### CODE REVIEW

**Comment by bebatut** [comment]
@EngyNasr could you have a look on the tests? Thanks a lot

File: workflows/microbiome/gene-based-pathogen-identification/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## [0.2] 2024-06-28
```
Comment (bebatut) [inline workflows/microbiome/gene-based-pathogen-identification/CHANGELOG.md:3]:
```suggestion
## [0.1.1] 2024-06-28
```

File: workflows/microbiome/gene-based-pathogen-identification/Gene-based-Pathogen-Identification.ga [hunk 2]
```diff
@@ -111,7 +111,7 @@
     ],
     "format-version": "0.1",
     "license": "MIT",
-    "release": "0.1",
+    "release": "0.2",
```
Comment (bebatut) [inline workflows/microbiome/gene-based-pathogen-identification/Gene-based-Pathogen-Identification.ga:114]:
```suggestion
    "release": "0.1.1",
```

**Comment by EngyNasr** [comment]
> @EngyNasr could you have a look on the tests? Thanks a lot

there is not even an error message to understand, i can only see that the updated tool version named Replace is in state error, I tried it locally against my galaxy instance it works, will check again now maybe I find something

**Comment by lldelisle** [comment]
@bebatut should I delete the branch to allow new updates?

**Pull request closed without merging**